client/me/purchases/cancel-purchase/refund-information.jsx

```diff
@@ -33,30 +33,33 @@ const CancelPurchaseRefundInformation = ( { purchase, includedDomainPurchase } )
 			if ( includedDomainPurchase && isDomainMapping( includedDomainPurchase ) ) {
 				text = [
 					i18n.translate(
-						'This plan includes the custom domain mapping for %(mappedDomain)s, normally a %(mappingCost)s purchase. ' +
-							'The domain will not be removed along with the plan, to avoid any interruptions for your visitors. ',
+						'This plan includes mapping for the domain %(mappedDomain)s. ' +
+						"Cancelling will remove all the plan's features from your site, including the domain.",
 						{
 							args: {
 								mappedDomain: includedDomainPurchase.meta,
-								mappingCost: includedDomainPurchase.priceText,
+								wordpressSiteUrl: purchase.domain,
```

Comment:
We should remove this `wordpressSiteUrl` argument since it's not used in the string above.